content/en/docs/concepts/security/service-accounts.md

```diff
@@ -189,17 +194,21 @@ to authenticate to the Kubernetes API server, and to any other system where a
 trust relationship exists. Depending on how the token was issued
 (either time-limited using a `TokenRequest` or using a legacy mechanism with
 a Secret), a ServiceAccount token might also have an expiry time, an audience,
-and a time after which the token *starts* being valid. When a process running
-in a Pod attempts to communicate with the Kubernetes API server, it adds an
-`Authorization: Bearer <token>` header to the HTTP request. The API server
-checks the validity of the bearer token as follows:
+and a time after which the token *starts* being valid. When a client that is
+acting as a ServiceAccount tries to communicate with the Kubernetes API server,
+the client includes an `Authorization: Bearer <token>` header with the HTTP
+request. The API server checks the validity of that bearer token as follows:
 
 1. Check the token signature.
 1. Check whether the token has expired.
-1. Checks whether object references in the token claims are currently valid
+1. Check whether object references in the token claims are currently valid.
 1. Check whether the token is currently valid.
 1. Check the audience claims.
 
+The TokenRequest API produces _bound tokens_ for a ServiceAccount. This
```

Comment:
bound or bounded?

Reply:
Bound is right - the tokens have been bound to an object. Bounded would mean that they had a defined scope, whereas we're talking about the existence of an object-token binding. Validity of the token is conditional on the continued existence of the object that the token is bound to.

Reply:
OK. I was thinking of the expiry time of the token which is bounded.